From d45fb976f46600554e2db02ce5d16d1dff84d106 Mon Sep 17 00:00:00 2001
From: Bjorn Andersson <quic_bjorande@quicinc.com>
Date: Mon, 9 Jan 2023 20:20:04 -0800
Subject: soc: qcom: ramp_controller: Make things static

The five msm8976_cfg_* objects ought to be static, as reported by LKP
and sparse, fix this.

drivers/soc/qcom/ramp_controller.c:235:27: sparse: sparse: symbol 'msm8976_cfg_dfs_sid' was not declared. Should it be static?
drivers/soc/qcom/ramp_controller.c:246:27: sparse: sparse: symbol 'msm8976_cfg_link_sid' was not declared. Should it be static?
drivers/soc/qcom/ramp_controller.c:250:27: sparse: sparse: symbol 'msm8976_cfg_lmh_sid' was not declared. Should it be static?
drivers/soc/qcom/ramp_controller.c:256:27: sparse: sparse: symbol 'msm8976_cfg_ramp_en' was not declared. Should it be static?
drivers/soc/qcom/ramp_controller.c:262:27: sparse: sparse: symbol 'msm8976_cfg_ramp_dis' was not declared. Should it be static?

Fixes: a723c95fa137 ("soc: qcom: Add Qualcomm Ramp Controller driver")
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Bjorn Andersson <quic_bjorande@quicinc.com>
Reviewed-by: AngeloGioacchino Del Regno <angelogioacchino.delregno@collabora.com>
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
Link: https://lore.kernel.org/r/20230110042004.2378444-1-quic_bjorande@quicinc.com
---
 drivers/soc/qcom/ramp_controller.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/soc/qcom/ramp_controller.c b/drivers/soc/qcom/ramp_controller.c
index 93652b07e812d..dc74d2a19de2b 100644
--- a/drivers/soc/qcom/ramp_controller.c
+++ b/drivers/soc/qcom/ramp_controller.c
@@ -232,7 +232,7 @@ static const struct regmap_config qrc_regmap_config = {
 	.fast_io = true,
 };
 
-const struct reg_sequence msm8976_cfg_dfs_sid[] = {
+static const struct reg_sequence msm8976_cfg_dfs_sid[] = {
 	{ 0x10, 0xfefebff7 },
 	{ 0x14, 0xfdff7fef },
 	{ 0x18, 0xfbffdefb },
@@ -243,23 +243,23 @@ const struct reg_sequence msm8976_cfg_dfs_sid[] = {
 	{ 0x2c, 0x8102 }
 };
 
-const struct reg_sequence msm8976_cfg_link_sid[] = {
+static const struct reg_sequence msm8976_cfg_link_sid[] = {
 	{ 0x40, 0xfc987 }
 };
 
-const struct reg_sequence msm8976_cfg_lmh_sid[] = {
+static const struct reg_sequence msm8976_cfg_lmh_sid[] = {
 	{ 0x30, 0x77706db },
 	{ 0x34, 0x5550249 },
 	{ 0x38, 0x111 }
 };
 
-const struct reg_sequence msm8976_cfg_ramp_en[] = {
+static const struct reg_sequence msm8976_cfg_ramp_en[] = {
 	{ 0x50, 0x800 }, /* pre_en */
 	{ 0x50, 0xc00 }, /* en */
 	{ 0x50, 0x400 }  /* post_en */
 };
 
-const struct reg_sequence msm8976_cfg_ramp_dis[] = {
+static const struct reg_sequence msm8976_cfg_ramp_dis[] = {
 	{ 0x50, 0x0 }
 };
 
-- 
cgit 

